src/mds/layout/mobile/headers/BasicHeader.jsx

```diff
@@ -20,7 +21,9 @@ const BasicHeader = ({
         ) : (
           <EmptyBox />
```

Comment:
`EmptyBox`를 리턴하지 않아도 괜찮지 않을까요?

```suggestion
{hasPrevButton && <PreviousArrowButton onClick={handlePreviousPage} />
}
```

Reply:
Container 에서  `justify-content: space-between;` 를 사용하여 간격을 조정하고 있어서 `EmptyBox`를  리턴하였습니다 !